app/src/main/res/values/strings.xml

```diff
@@ -31,12 +31,16 @@
 
     <string name="settings_cgpa_label">CGPA CALCULATOR</string>
     <string name="settings_in_progress_course">Include In-Progress Courses</string>
-    <string name="settings_in_progress_description">If toggled on, courses without a Final Grade specified will be included in your CGPA calculations on the CGPA Calculator page.</string>
+    <string name="settings_in_progress_description">If toggled on, courseEntity without a Final Grade specified will be included in your CGPA calculations on the CGPA Calculator page.</string>
```

Comment:
This should have stayed as "courses", instead of being converted to "courseEntity"!